_data/chains/eip155-3141.json

```
@@ -1,19 +1,44 @@
 {
-  "name": "Filecoin - Buildernet",
+  "name": "Filecoin - Hyperspace testnet",
   "chain": "FIL",
-  "status": "incubating",
-  "rpc": [],
-  "faucets": [],
+  "icon": "filecoin",
+  "rpc": ["https://hyperspace.node.glif.io/rpc/v0"],
+  "faucets": ["https://hyperspace.filtest.network/#faucet"],
   "nativeCurrency": {
-    "name": "filecoin",
-    "symbol": "FIL",
+    "name": "testnet filecoin",
+    "symbol": "tFIL",
     "decimals": 18
   },
   "infoURL": "https://filecoin.io",
-  "shortName": "filecoin-buildernet",
-  "icon": "filecoin",
+  "shortName": "filecoin-hyperspace",
   "chainId": 3141,
   "networkId": 3141,
   "slip44": 1,
-  "explorers": []
+  "explorers": [
+    {
+      "name": "Filfox - Hyperspace",
+      "url": "https://hyperspace.filfox.info/en",
+      "standard": "none"
+    },
+    {
+      "name": "Glif Explorer - Hyperspace",
+      "url": "https://explorer.glif.io/?network=hyperspace",
+      "standard": "EIP3091"
```

this one also does not really support EIP3091 - e.g. blocks and tokens seem to be unsupported. Maybe https://github.com/ChainAgnostic/CAIPs/pull/200 can help in the future - but for now we should set to none to be able to merge 

```suggestion
      "standard": "none"
```